test/relations.test.js

```diff
@@ -835,6 +835,23 @@ describe('relations', function() {
         });
       });
 
+      context('findById with filter include that contains scope with string filter fields', function() {
+        it('should be accepted and convert fields to array.', function(done) {
+          var includeFilter = {include: {relation: 'patients', scope: {fields: 'name'}}};
+          var physicianId = physician.id;
+          Physician.findById(physicianId, includeFilter, function(err, ch) {
+            should.not.exist(err);
+            should.exist(ch);
+            ch.id.should.eql(physicianId);
+            should.exist(ch.patients);
```

Comment:
add another assertion to actually check for the array instance, something like `result.patients().should.be.an.instanceOf(Array);`